### PR TITLE
nwjs-ffmpeg-prebuilt: init at 0.89.0

### PR DESCRIPTION
**Pull request by ilya-epifanov**
## Description of changes

Ffmpeg prebuilt for nwjs codec support.
Required for some software, including:
```
inav-blackbox-explorer
```

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by ilya-epifanov** [comment]
required for https://github.com/NixOS/nixpkgs/pull/310813

File: pkgs/development/tools/nwjs-ffmpeg-prebuilt/default.nix
Comment (eclairevoyant) [inline pkgs/development/tools/nwjs-ffmpeg-prebuilt/default.nix:35]:
```suggestion
```

File: pkgs/development/tools/nwjs-ffmpeg-prebuilt/default.nix
Comment (eclairevoyant) [inline pkgs/development/tools/nwjs-ffmpeg-prebuilt/default.nix:30]: `nativeBuildInputs` can be removed entirely

File: pkgs/development/tools/nwjs-ffmpeg-prebuilt/default.nix
Comment (eclairevoyant) [inline pkgs/development/tools/nwjs-ffmpeg-prebuilt/default.nix:20]: were these hashes checked properly? these are the same hash.
Also please use SRI hash format.

File: pkgs/development/tools/nwjs-ffmpeg-prebuilt/default.nix
Comment (eclairevoyant) [inline pkgs/development/tools/nwjs-ffmpeg-prebuilt/default.nix:42]:
```suggestion
  meta = {
```

trcaking issue: #292468

**Review by eclairevoyant — DISMISSED** [review]
Thanks for the contribution. Some feedback below:

**Comment by MikaelFangel** [comment]
Partly fixes: #219662

File: pkgs/development/tools/nwjs-ffmpeg-prebuilt/default.nix
Comment (ilya-epifanov) [reply pkgs/development/tools/nwjs-ffmpeg-prebuilt/default.nix]: I didn't mean to :) I just used the nwjs package as a template. But now that I think of it, I'm not sure if I can be a reliable maintainer. Can I leave you as a maintainer?

**Comment by ilya-epifanov** [comment]
@eclairevoyant @MikaelFangel I've addressed all the comments. Huge thanks for the feedback!

File: pkgs/development/tools/nwjs-ffmpeg-prebuilt/default.nix
Comment (MikaelFangel) [reply pkgs/development/tools/nwjs-ffmpeg-prebuilt/default.nix]: Sure, I'll maintain it (:

File: pkgs/development/tools/nwjs-ffmpeg-prebuilt/default.nix
Comment (ilya-epifanov) [reply pkgs/development/tools/nwjs-ffmpeg-prebuilt/default.nix]: `unzip` is actually required to unpack the sources

**Comment by MikaelFangel** [comment]
You also need to change some things in this PR before it is ready to merge.
 - You need to change the pr title to: nwjs-ffmpeg-prebuilt: init at 0.87.0
 - The commit adding you as a maintainer to Nixpkgs needs to come before the package commit
 - Then the commit adding the package needs to have the commit message: nwjs-ffmpeg-prebuilt: init at 0.87.0
 - The two before mentioned commits should be the only commits in this PR, so you need to squash the last commit with the commit adding the package.

And thank you for your contribution to Nixpkgs. (:

Please refer to the guidelines if in doubt: 
 - https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#commit-conventions
 - https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#how-to-create-pull-requests
 - https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#pull-request-template

**Review by MikaelFangel — APPROVED** [review]

**Comment by MikaelFangel** [comment]
Note: nwjs is currently at version 0.89.0

**Review by JohnRTitor — APPROVED** [review]
Builds for me locally and I just applied minor `nixfmt-rfc-style`, and updated it

**Pull request merged**
